index.html

```diff
@@ -471,13 +471,16 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         through a transformation function (<b>iteratee</b>). The <tt>iteratee</tt>
         is passed three arguments: the <tt>value</tt>, then the <tt>index</tt>
         (or <tt>key</tt>) of the iteration, and finally a reference to the entire
-        <tt>list</tt>.
+        <tt>list</tt>. Selective <b>Underscore</b> functions marked as <tt>guarded</tt>
+        can be used as an <tt>iteratee</tt> to <b>_.map</b>.
```

Comment:
Imagine Underscore get's a `_.trim` or other like method. `_.filter(arrayOfStrings, _.trim)`. It's better to be clear about the guard rather than boxing in. You could do this by simply saying "used as an `iteratee` to `_.map` and the like."